doc/version_history.rst

```diff
@@ -6,6 +6,10 @@
 Version History
 ##################
 
+.. _lsst.ts.mtdomegui-0.5.2:
+
+* Report the louver control system to be Enabled once there is the connection.
```

Comment:
`louver control system` -> `louvers control system`

Reply:
Improved in the commit: [32ab450](https://github.com/lsst-ts/ts_mtdomegui/pull/21/commits/32ab4507e8234ce0049b5d5bc1c2cd1ce32abe09). Thanks!